composer.json

```diff
@@ -57,7 +57,7 @@
         "symplify/phpstan-rules": "^12.4.5",
         "symplify/rule-doc-generator": "^12.1.10",
         "symplify/vendor-patches": "^11.2",
-        "tomasvotruba/class-leak": "0.2.11",
+        "tomasvotruba/class-leak": "^0.2.11",
```

Comment:
^0.2.13

Reply:
it does not make a difference but sure why not ;)